scripts/gcs/a6000/picture_taker.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+while [ true ]
+do
+  fn=$(date +%s)
+  gphoto2 --capture-image-and-download --filename image-$fn.jpg
+  mv image-$fn.jpg imgs/
+  sleep 5
```

Comment:
Please make 5 a variable at the top of this script with an easy to understand name so others understand that this is the picture capture interval. It would also be nice if you timed how long it took to actually take the image and move it and added that information as a comment here so that we can time this correctly in the final setup.